README.md

```diff
@@ -299,11 +299,11 @@ The following settings can be optionally set to customize the node labels and ta
   * Example user data for setting up labels and taints:
     ```
     [settings.kubernetes.node-labels]
-    label1 = "foo"
-    label2 = "bar"
+    "label1" = "foo"
+    "label2" = "bar"
     [settings.kubernetes.node-taints]
-    dedicated = "experimental:PreferNoSchedule"
-    special = "true:NoSchedule"
+    "dedicated" = "experimental:PreferNoSchedule"
+    "special" = "true:NoSchedule"
```

Comment:
In some of the other settings descriptions, we have a note like this:
```
Remember to quote <X> (since they often contain ".") and to quote all values.
```

It might be useful to add that to the end of the `node-labels` and `node-taints` items, just to clarify why the quotes show up in the example.

Reply:
WDYT about keeping quote in all documentation and recommend users use them in all cases. You probably need to support the unquoted variant for backwards compat, but maybe best to leave that undocumented as undesired behavior.

Reply:
i.e., removing the need for any reminder of adding the quotes and simply show them in all examples.

Reply:
Always using quotes would go against the spirit of the TOML standard, where quotes are for special cases; we try to keep things simple for the majority of cases that don't need quoting.

Reply:
I'm all about following TOML standards, but the problem is that 99% of kubernetes labels and taints are domains, and include special characters, so users (like myself) will trip over this.

Reply:
I think we agree that quotes are good in this case :)  I thought you were saying to quote keys for all TOML examples in the rest of the documentation, too.  We use TOML in many places, and very little of it requires quoting.

Reply:
We're on the same page :). Any other spots I might've missed?

Reply:
I don't think so!  Adding the comment above would help for consistency with other areas where we need quoting, but other than that I think we're good.

Reply:
![image](https://user-images.githubusercontent.com/2585992/116760922-250d0e80-a9cb-11eb-967a-14bfa5d874e9.png)